tools/lib/capitalization.py

```diff
@@ -89,6 +89,7 @@
     # these are used as topics
     r"^new streams$",
     r"^stream events$",
+    r"^new organizations$",
```

Comment:
This is no longer needed if the string isn't tagged for translation anymore, right?

Reply:
Yes, thanks for pushing that update!